tools/installer/debian/build.sh

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash
+#
+# Copyright (c) 2012 The Chromium Authors. All rights reserved.
+# Copyright (c) 2015 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file is a tuned-down version of the scripts used to build Chrome Debian
+# packages. See src/chrome/installer/linux/debian for the original.
+
+set -e
+set -o pipefail
+if [ "$VERBOSE" ]; then
+  set -x
+fi
+set -u
+
+# Create the Debian changelog file needed by dpkg-gencontrol. This just adds a
+# placeholder change, indicating it is the result of an automatic build.
+# TODO(mmoss) Release packages should create something meaningful for a
+# changelog, but simply grabbing the actual 'svn log' is way too verbose. Do we
+# have any type of "significant/visible changes" log that we could use for this?
+gen_changelog() {
```

Comment:
We used to use git log to generate changelog: https://github.com/seanlong/crosswalk-package-tool/blob/master/lib/DebProject.js#L44